README.md

```diff
@@ -78,33 +83,70 @@ In practice, **entanglement** arises when the operation of a gate on a qubit is
+
+- [Qubit nodes](https://github.com/node-red-quantum/node-red-contrib-quantum/wiki/Qubit-Nodes) (blue) are used to keep track and manage qubits, such as resetting them. The 'Qubit' node receives qubits as input but does not execute any operation on it. It can be used to identify qubits, by printing some text under the node, or to rearrange the position of qubits into the editor.
+
+- [Quantum Gate nodes](https://github.com/node-red-quantum/node-red-contrib-quantum/wiki/Quantum-Gate-Nodes) correspond to quantum operations. When a qubit flow through a quantum gate node, the operation is executed on the qubit. There are 2 type of quantum gates:
+  - [Singe qubit gates](https://github.com/node-red-quantum/node-red-contrib-quantum/wiki/Quantum-Gate-Nodes/#single-qubit-quantum-gate) (purple) receive a qubit, execute an operation, and output the qubit. The 'NOT Gate' flips the state of the qubit (0 &#8594; 1 & 1 &#8594; 0).
+  - [Multi qubits gates](https://github.com/node-red-quantum/node-red-contrib-quantum/wiki/Quantum-Gate-Nodes#multi-qubits-quantum-gates) (red) receive a certain number of qubits, execute a **control operation** and output the qubits. Control operations correspond to the conditional execution of a single quantum gate. If the 'control' qubits are in the correct state, then the gate is applied on a 'target' qubit. The 'CNOT Gate', operating on 2 qubits, applies a NOT gate on the 'target' qubit if the 'control' qubit is in the '1' state.
+
+- [Output nodes](https://github.com/node-red-quantum/node-red-contrib-quantum/wiki/Output-Nodes) (grey) are used to process the quantum circuit and generate an output. All qubits of the quantum circuit must be connected to the same output node for it to work. The output format varies between each node, please refer to the documentation. The 'Local Simulator' node simulates a quantum computer using the local Qiskit library. It runs the quantum circuit and outputs the results.
+
+### Rules
+1. **Do not duplicate qubits**  
+   There should always be 1 instance of each qubit at all times.  
+   A node can take multiple qubits as input but only one wire should be connected to each output. To do so, always input as many qubits as the node outputs.
 
-More information on quantum nodes can be found in the [Node-RED Quantum wiki](https://github.com/node-red-quantum/node-red-contrib-quantum/wiki/Building-Quantum-Circuits).
+2. **Only 1 quantum cicuit per Node-RED tab**  
+   A maximum of 1 quantum circuit must be used on each Node-RED tab.  
+   If an issue occurs, please refresh the page.
+
+3. **Wait for quantum circuit to execute**  
+   Do not double-click the inject button, wait for the circuit to have finished executing before runnning it again.  
+   If an issue occurs, please refresh the page.
```

Comment:
Are these rules true? I think rule 2 is no longer an issue because the global state manager creates an isolated state object separate from other circuits, even if they are in the same flow. With rule 3, what kind of errors can occur if a quantum circuit is executed before finishing?

P.s. I just noticed there is a typo for the word "circuit" in rule 2.

Reply:
Hey Louis, 

Makes sense for rule 2, I'll get rid of it then.

The issue with rule 3 is if the user triggers the flow while it was in execution. The nodes waiting for a certain amount of qubits (e.g. output nodes) could only have received half of the qubits in the first execution and therefore not be reset. In that case, the second half of qubits will come from the second execution which leads to qubit duplications and inconistencies.